contracts/strategies/mstable/IdleMStableStrategy.sol

```diff
@@ -255,24 +265,20 @@ contract IdleMStableStrategy is Initializable, OwnableUpgradeable, ERC20Upgradea
     }
 
     /// @notice Claim governance tokens
-    /// @param startRound Start Round from which the Governance tokens must be claimed
     /// @param endRound End Round from which the Governance tokens must be claimed
-    function claimGovernanceTokens(uint256 startRound, uint256 endRound) public onlyOwner {
-        _claimGovernanceTokens(startRound, endRound);
+    function claimGovernanceTokens(uint256 endRound) public onlyOwner {
+        (, , uint256 _endRound) = vault.unclaimedRewards(address(this));
+        require(endRound > rewardLastRound, "End round should more than the last claimed round ");
```

Comment:
are they doing some of these checks too? what happens if `endRound > _endRound` ?